user-data-store-server/pom.xml

```diff
@@ -93,6 +93,12 @@
             <artifactId>postgresql</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.microsoft.sqlserver</groupId>
+            <artifactId>mssql-jdbc</artifactId>
+            <version>${mssql-jdbc.version}</version>
```

Comment:
We use the implicit version from Spring Boot parent for postgres. May we use it for MS SLQ as well?

Reply:
Ok.